src/commands/docker-install-cli.yml

```diff
@@ -1,3 +1,5 @@
+---
+
 description: Configure Docker remote registry [AWS ECR].
 
 steps:
```

Comment:
🐼 

Maybe a good occasion to update to latest docker engine available in CircleCI
(currently locally 24.0.5)